pkg/apis/kf/v1alpha1/utils.go

```diff
@@ -43,6 +43,7 @@ func PropagateCondition(manager apis.ConditionManager, destination apis.Conditio
 		manager.MarkTrue(destination)
 	case source.IsUnknown():
 		manager.MarkUnknown(destination, source.Reason, source.Message)
+		return false
```

Comment:
nit: Is this necessary? The latter line (`return source.IsTrue()`) will also return `false`, right?

Reply:
Surely, removed. A wrong assertion messed my mind before.